evadb/binder/create_index_statement_binder.py

```diff
@@ -17,12 +17,32 @@
 from evadb.catalog.catalog_type import NdArrayType, VectorStoreType
 from evadb.parser.create_index_statement import CreateIndexStatement
 from evadb.third_party.databases.interface import get_database_handler
+from evadb.expression.tuple_value_expression import TupleValueExpression
+from evadb.expression.function_expression import FunctionExpression
+from evadb.binder.binder_utils import extend_star, create_row_num_tv_expr
 
 
 def bind_create_index(binder: StatementBinder, node: CreateIndexStatement):
     binder.bind(node.table_ref)
-    if node.function:
-        binder.bind(node.function)
+
+    func_project_expr = None
+
+    # Extend projection list.
+    project_expr_list = []
+    for project_expr in node.project_expr_list:
+        if isinstance(project_expr, TupleValueExpression) and project_expr.name == "*":
```

Comment:
Can we create index on multiple columns now?

Reply:
Not really. I use this to project the raw table column if index is built on table directly. I use `*` just for simplicity for now.

Reply:
If we have a query `CREATE INDEX test_index ON myTable (myCol);`, can the `project_expr_list` be `[TupleValueExpression(myCol)]` instead of `[TupleValueExpression(*)]`?

Reply:
Fixed.